tests/timeago.test.js

```diff
@@ -0,0 +1,119 @@
+var should = require('should');
+var levels = require('../lib/levels');
+var times = require('../lib/times');
+
+describe('timeago', function ( ) {
+
+  var timeago = require('../lib/plugins/timeago')();
+  var delta = require('../lib/plugins/delta')();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['delta' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1128648795/issues/source?bid=2452379&fileBranchId=2903028#l8)